test/tests/cache.test.js

```diff
@@ -198,13 +203,13 @@ describe('Handling cache set/get', () => {
     const cacheHtml = cacheManager.getCache('html')
     cacheHtml.reset()
 
-    request.get(`${uri}/2017/12/01/one-item`, () => {
-      expect(cacheHtml.keys()).to.have.length(1)
-      request.get(`${uri}/2017/12/01/another-item`, () => {
-        expect(cacheHtml.keys()).to.have.length(2)
-        request.get(`${uri}/2017/12/01/another-another-item`, () => {
-          expect(cacheHtml.keys()).to.have.length(2)
-          expect(cacheHtml.keys()).to.not.include('2017/12/01/one-item')
+    request.get(`${uri}/2017/12/01/one-item`, async () => {
+      expect(await cacheHtml.keys()).to.have.length(1)
```

Comment:
I used the previous test setup here because ~_I like to mix things up_~ I grokked this test in this way straight away.